app/services/DomainService.scala

```diff
@@ -44,14 +44,15 @@ class DomainService {
         .map(_.map(ds => toDomain(ds.dailyStatement))).getOrElse(Seq.empty))
   }
 
+  import domain.Declaration
   private def toDomain(declaration: DeclarationDetail): domain.Declaration = {
     Declaration(declaration.declarationID,
       declaration.importerEORINumber,
       declaration.declarantEORINumber,
       declaration.declarantReference,
       declaration.postingDate,
       declaration.amount,
-      Nil)
+      declaration.taxGroups.map(container => toDomainTaxGroup(container.taxGroup)))
```

Comment:
Could you please write tests for this change i.e. for toDomainSummary method? If it takes time then leave it.